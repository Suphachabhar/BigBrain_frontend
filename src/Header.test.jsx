import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import { Header } from './components/Header.styles';

describe('Header', () => {
  // test the header component
  it('uses custom title', () => {
    const title = 'A custom title';
    const header = shallow(<Header title={title} />);
    expect(header.text()).toBe(title);
  });

  it('renders with provided title (snapshot)', () => {
    const header = renderer.create(<Header title={'title'} />).toJSON;
    expect(header).toMatchSnapshot();
  });
});
