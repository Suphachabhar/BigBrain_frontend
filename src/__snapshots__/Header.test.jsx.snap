// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders with provided title (snapshot) 1`] = `[Function]`;
